src/modules/review_sentiments.py

```diff
@@ -0,0 +1,65 @@
+from typing import Dict, Any
+from .base_module import BaseModule
+from data_loader import Data
```

Comment:
Should be the path, not ..data_loader -?

Reply:
The program is run from inside the src folder and it is importi0ng from the file `src/data_loader.py`.